src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  
+
+  &__card {
+    margin-bottom: 47px;
+  }
+ }
+
+.catalog__container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: $wrapper-width;
```

Comment:
this width use only one time, it's not necessary to use variables for it.
And if you remove flex from the `.catalog` add `margin 0 auto` here, justify-content remove to.